lib/dialects/postgres/query-generator.js

```diff
@@ -900,6 +906,36 @@ class PostgresQueryGenerator extends AbstractQueryGenerator {
   dropForeignKeyQuery(tableName, foreignKey) {
     return `ALTER TABLE ${this.quoteTable(tableName)} DROP CONSTRAINT ${this.quoteIdentifier(foreignKey)};`;
   }
+
+  /**
+   * Quote identifier in sql clause
+   *
+   * @param {string} identifier
+   * @param {boolean} force
+   *
+   * @returns {string}
+   */
+  quoteIdentifier(identifier, force) {
+    const optForceQuote = force || false;
+    const optQuoteIdentifiers = this.options.quoteIdentifiers === false ? false : true;
```

Comment:
I tend to believe that this is the same as `this.options.quoteIdentifiers !== false`